nanshe_ipython.ipynb

```diff
@@ -1155,14 +1157,28 @@
         "        title=title,\n",
         "        div=indent(div, 8),\n",
         "        script=indent(script, 8),\n",
-        "        cdn=indent(bokeh.resources.CDN.render(), 8),\n",
+        "        cdn=indent(cdn, 8),\n",
         "    )\n",
         "\n",
         "    with io.open(filename, \"w\") as fh:\n",
         "        fh.write(html_cont)\n",
         "\n",
         "script, div = be.components(plot_group)\n",
-        "write_html(data_proj_html, os.path.splitext(data_proj_html)[0], div, script)"
+        "cdn = bokeh.resources.CDN.render()\n",
+        "cdn += \"\"\"\n",
+        "\n",
+        "<script type=\"text/javascript\" src=\"https://cdn.jsdelivr.net/pako/1.0.4/pako.min.js\"></script>\n",
```

Comment:
This pulls in all of pako. In reality, we only need the portion related to decompressing the data. As such, we can change this to https://cdn.jsdelivr.net/pako/1.0.4/pako_inflate.min.js , which would be a little smaller to download. Though I don't expect this matters much as both are on the order of 10KB.